addon/v1alpha1/types_managedclusteraddon.go

```diff
@@ -245,7 +250,23 @@ const (
 	// after the addon is deleted.
 	DeletionOrphanAnnotationKey = "addon.open-cluster-management.io/deletion-orphan"
 
-	// HostedManifestLocationLabelKey is the label key for indicating the cluster that the manifest should be deployed in Hosted
-	// mode.
+	// HostedManifestLocationLabelKey is the label key for indicating the cluster that the manifest should be deployed
+	// in Hosted mode.
+	// Deprecated, will be removed in the future release, please use annotation HostedManifestLocationAnnotationKey from v0.10.0.
 	HostedManifestLocationLabelKey = "addon.open-cluster-management.io/hosted-manifest-location"
+
+	// HostedManifestLocationAnnotationKey is the annotation key for indicating the cluster that the manifest should be
+	// deployed in Hosted mode.
```

Comment:
should indicate this annotation is added on which resource?

Reply:
added